src/EPR.SubsidiaryBulkUpload.Application/Services/TableStorageProcessor.cs

```diff
@@ -0,0 +1,91 @@
+﻿using EPR.SubsidiaryBulkUpload.Application.Services.Interfaces;
+using EPR.SubsidiaryBulkUpload.Application.Services.Models;
+using Microsoft.Azure.Cosmos.Table;
+using Microsoft.Extensions.Logging;
+
+namespace EPR.SubsidiaryBulkUpload.Application.Services;
+
+public class TableStorageProcessor(
+    ILogger<CompaniesHousCsvProcessor> logger) : ITableStorageProcessor
+{
+    private const string CurrentIngestion = "Current Ingestion";
+    private const string LatestCHData = "Latest CH Data";
+    private const string Latest = "Latest";
+    private readonly ILogger<CompaniesHousCsvProcessor> _logger = logger;
+
+    public async Task WriteToAzureTableStorage(IEnumerable<CompanyHouseTableEntity> records, string tableName, string partitionKey, string connectionString, int batchSize)
+    {
+        // Connect to the local Azure Table Storage
```

Comment:
Comments should be removed to appease sonarqube

Reply:
Done.